### DISCUSSION TITLE
imagegd2(): GD2 image support has been disabled

### DISCUSSION THREAD

**Discussion by rybajz** [shivammathur/homebrew-php#701] [Q&A / Help] [answered]
Hi,
thank you for your great PHP sw.

I have problem with images. I use imagegd2($this->image); for manipulating images. But I am getting following error:
imagegd2(): GD2 image support has been disabled

This error appear on 7.4 and 8.0.
On 7.3 is everything ok.

Could you please help me solve this isssue?

Thanks

**Comment by shivammathur** [comment, marked as answer]
GD image formats are old and not used anymore, so they are disabled by default since `libgd 2.3.3`.
https://github.com/libgd/libgd/issues/428

Therefore they are also disabled in the Homebrew's build of `libgd`, and that is used in PHP 7.4 and above.
So support for this won't be added.

**Reply by rybajz** [reply]
but I use photo in jpg which I have createted yesterday. Is there some solution?

**Reply by shivammathur** [reply]
`imagegd2` function outputs images in gd2 format.

Use `imagejpeg()` for jpg
https://www.php.net/manual/en/function.imagejpeg.php
